tests/app/v2/notifications/test_notification_schemas.py

```diff
@@ -26,7 +24,7 @@
 }
 
 
-@pytest.mark.parametrize("input", [valid_get_json, valid_get_with_optionals_json])
+@pytest.mark.parametrize("input", [{}, valid_get_with_optionals_json])
```

Comment:
Hmm while the `valid_get_json` was just an empty dictionary, labeling it as such with a variable name might be helpful to indicate that this was a valid JSON. I'll let you decide though, I am not familiar with the test code but I do like verbose naming that might clear out what's the goal of the `input`.

Reply:
Good point! I will revert this change and add a type definition